OverlayPlugin.Core/EventSources/EnmityEventSource.cs

```diff
@@ -203,6 +203,8 @@ protected override void Update()
 
                 var combatants = memory.GetCombatantList();
 
+                combatants.RemoveAll((c) => c.Type != ObjectType.PC && c.Type != ObjectType.Monster);
```

Comment:
How many combatants are there normally before removing them here? This is maybe just premature optimization, but wondering if makes sense to pass in a filter vs construct a list and then remove things from it.

I wonder also if maybe (in a future PR) we could pass in ObjectType filters to getCombatants here (or some other method).

Reply:
In-instance, I've seen only a couple extra combatants come back from this. There's significantly more in e.g. housing areas, but even there the impact is only a couple milliseconds of extra time to call.